files/default/github_pr_comment_trigger.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+
+# Takes a JSON payload from a Github PR issue webhook from stdin, reads the
+# comment, and if it matches `!bump (major|minor|patch)?` (defaults to
+# `minor`), merges the PR, bumps the Chef cookbook's metadata.rb, creates a
```

Comment:
I'm assuming people will accidentally forget to specify the bump level occasionally, and I figured that accidentally doing a minor-level bump for a patch is safer than doing a patch-level bump for a feature.

Maybe not having a default at all would be better.